pkg/cmd/pr/checks/checks.go

```diff
@@ -66,6 +66,13 @@ func NewCmdChecks(f *cmdutil.Factory, runF func(*ChecksOptions) error) *cobra.Co
 
 			Without an argument, the pull request that belongs to the current branch
 			is selected.
+
+			The --json flag outputs the check information in JSON format. It includes:
+
+			- 'bucket': Categorizes checks as "pass", "fail", "pending", "skipping", or "cancel".
+			- 'state': Provides the specific status like "SUCCESS", "FAILURE", "PENDING", etc.
+
+			The 'bucket' field simplifies statuses into categories, while 'state' gives the exact status.
```

Comment:
This is great! I think there's probably value in being a bit verbose on how `buckets` and `states` map, here, as I'm not sure it's documented elsewhere. I suggest we add something like this:

```suggestion
			The 'bucket' field simplifies statuses into categories, while 'state' gives the exact status:
			
			| bucket | state |
			| ------- | ------ |
			| pass | SUCCESS |
			| skipping | SKIPPED, NEUTRAL |
			| fail | ERROR, FAILURE, TIMED_OUT, ACTION_REQUIRED |
			| cancel | CANCELLED |
			| pending | EXPECTED, REQUESTED, WAITING, QUEUED, PENDING, IN_PROGRESS, STALE |
```

Alternatively, we could [link to the switch statement in the code](https://github.com/cli/cli/blob/dbcac5abc4ae8c20008ee25288f30a3aae0e0369/pkg/cmd/pr/checks/aggregate.go#L73-L87). What do you think would work best?

Reply:
I have 2 concerns to share:

1. Help usage should be properly backtick quoted as this affects web manual formatting of pages; see https://github.com/cli/cli/pull/8342 for more information

2. How does well does this table format render for in-line usage as well as man pages?

I don't know if we have tables within any command as this is GitHub Flavored Markdown, so I'd really ground this in the structure and format of existing help commands.

Reply:
Ah, good point about markdown syntax... 

It's not super clear to me what "properly backtick quoted" means. I see the example PR you've linked, but it's expansive and a bit hard to follow the conversation without context. Could you be more specific with an example about what you mean for "properly backtick quoted"?

Reply:
Yeah, let me create an actual `suggestion` comment but I think the following example from `gh api` showing how the long usage uses backticks to quote flags and keywords is expected rather than single or double quotes

https://github.com/cli/cli/blob/9e27af999ebd7e776ba5c2373371ed59a6eda096/pkg/cmd/api/api.go#L72-L125

Reply:
So am I correct in understanding that you're suggesting all the `"` be replaced with `` ` `` by using the formatted string, where `%[1]s` in the above example is replaced by `` "`" ``?

Also, to avoid bad table formatting a bulleted list works just as well, something like:

```
- `pass`: `SUCCESS`
- `skipping`: `SKIPPED`, `NEUTRAL`
- ...
```

Reply:
Essentially; the linked PR above was an effort to align all command help usage around single quotes, double quotes, no quotes to all be backticks / apostrophes.

The challenge is that these multi-line heredocs already use backticks, and the process of escaping backticks is kinda painful.  So we leverage a `Printf` trick with heredoc to replace `%[1]s` in this instance with backtick.